components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -584,7 +583,7 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_old,th,qv_old,qv,dt,qitot,qirim,nitot,birim,ss
     ! causing energy conservation errors.
     inv_exner = 1._rtype/exner        !inverse of Exner expression, used when converting potential temp to temp
     t       = th    *inv_exner    !compute temperature from theta (value at beginning of microphysics step)
-    t_old   = th_old*inv_exner    !compute temperature from theta (value at beginning of model time step)
+    t_old   = th_old*inv_exner    !compute temperature from theta (value at beginning of model time step), AaronDonahue: Now that we are removing ssat do we need t_old?
```

Comment:
we should definitely remove "_old" stuff.

Reply:
Sounds good, I agree.